lib/spack/spack/ci.py

```diff
@@ -551,10 +551,10 @@ def generate_gitlab_ci_yaml(
     env,
     print_summary,
     output_file,
+    *,
```

Comment:
is this a typo?

Reply:
No, this will force all the arguments afterwards to be keyword only arguments. This means at the calling point the argument name must be spelled out.

Reply:
Relevant PEP: https://peps.python.org/pep-3102/

Reply:
oh neat!